### DISCUSSION TITLE
Apparently spurious relicensing

### DISCUSSION THREAD

**Discussion by Kodiologist** [Tyrrrz/DiscordChatExporter#1146] [General]
### Version

master HEAD (3647142a96f8f41193267cf4eb354d585e4efe60)

### Flavor

CLI (Command-Line Interface)

### Platform

n/a

### Steps to reproduce

n/a

### Details

In 55270ad6d3dd2628af698d8bf06d3ef034b8f656, you deleted the text of the GPL 3 in `License.txt` and replaced it with the text of the MIT license. There's no obstacle to relicensing your own work in this fashion, but some of the code was contributed by other people. They still hold the copyright to their own contributions unless they explicitly assigned it to you, and the GPL imposes restrictions that make reuse under the MIT license a license violation. The upshot is that if you want the whole codebase usable under MIT, you either need to get contributors to agree to relicense under MIT or back out their changes. Either of these could be a huge hassle, so in your situation, I'd just rollback the license change.

### Checklist

- [X] I have looked through existing issues to make sure that this bug has not been reported before
- [X] I have provided a descriptive title for this issue
- [X] I have made sure that that this bug is reproducible on the latest version of the application
- [X] I have provided all the information needed to reproduce this bug as efficiently as possible
- [ ] I have sponsored this project

**Comment by 96-LB** [comment]
I personally agree to allow all my contributions to this project to be relicensed.

**Comment by ciroH** [comment]
This issue needs more attention, as it is clearly a violation of the GPL license, i agree.

Let's hope that Tyrrrz backs out with this license change as soon as possible.

**Comment by Tyrrrz** [comment]
Good point. I'll message the past contributors for their retroactive permission.
